tests/integration/pilot/webhook_test.go

```diff
@@ -83,8 +83,10 @@ func TestWebhook(t *testing.T) {
 			if t.Settings().Revisions.Default() != "" {
 				revision = t.Settings().Revisions.Default()
 			}
-			verifyRejectsInvalidConfig(t, revision, true)
-			verifyRejectsInvalidConfig(t, "", true)
+			if !cluster.IsRemote() {
```

Comment:
I was thinking the remote cluster didn't need to test validation, since if I didn't specify remote cluster, `integ-pilot-istiodremote_istio` and `integ-pilot-istiodremote-mc_istio` failed with TestValidation function.

Reply:
I think we should still have validation in these modes. The test may be testing the wrong thing but we should have _some_ validation. Not an expert here though

Reply:
cc @frankbu

Reply:
I can't see how `IsRemote()` would ever return true here, since this test is skipped for anything other than a single primary cluster topology: https://github.com/istio/istio/blob/f5a906e02c64c38500afc2d231c2f74d9ac36574/tests/integration/pilot/webhook_test.go#L40

That said, I think that maybe it could/should also be run in the istiodremote topology. In that case I think the check might need to be `if cluster.IsConfig()` instead of `if !cluster.IsRemote()`

Reply:
Thanks for the help, I'm going to look into it and have a try

Reply:
Not sure if the clusters' orders will change here, but after I revised `t.Clusters().Default()` (first cluster in the list) to explicitly be the config cluster, the tests passed.